src/assets/languages/english.json

```diff
@@ -238,8 +240,42 @@
         "heading": "Prevention",
         "stopTheBreed": {
           "heading": "Eliminate all the mosquito breeding places by clicking over them ",
-          "alerts" : {
-            "title" : "Congratulations!! <br> You eliminated a "
+          "alerts": {
+            "title": "Congratulations!! <br> You eliminated a "
+          }
+        },
+        "stridesAgainst": {
+          "heading": "Stride Against Malaria",
+          "howtoplay": {
+            "instructions": {
+              "title": "How to play?",
+              "ins1": "Click over the man to make it move.",
```

Comment:
Change ins1 to "Click on the person to move and reach the objects placed on the board".
Also add the same styling as it is in the locked stage